src/utils/gaa.js

```diff
@@ -302,10 +302,12 @@ let GoogleUserDef;
 
 /**
  * Returns true if the URL contains fresh Google Article Access (GAA) params.
+ * @param {string=} opt_urlParams
  * @return {boolean}
  */
-export function urlContainsFreshGaaParams() {
-  const params = parseQueryString(GaaMeteringRegwall.location_.search);
+export function urlContainsFreshGaaParams(opt_urlParams = '') {
+  const urlParams = opt_urlParams || GaaMeteringRegwall.location_.search;
```

Comment:
```suggestion
  const queryString = optionalQueryString || GaaMeteringRegwall.location_.search;
```

Reply:
Actually, I'm going to send you a PR that refactors this method a bit... please feel free to hold off on changing it

Reply:
Here's the PR! https://github.com/subscriptions-project/swg-js/pull/1480

Reply:
Thanks! Resolving since function is updated.